CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.2] - 2020-08-24
+
+ * `[fixed]` Fix overflow bug with high flow rates (#19)
```

Comment:
I think it's #20

Reply:
should we indicate at what flow rate the overflow would have happened? @rnestler

Reply:
> should we indicate at what flow rate the overflow would have happened?

Yeah, maybe add something like "... with flow rates higher than $FOO"